server_api.py

```diff
@@ -609,12 +610,10 @@ def store_thumbnail_image(file_path,thumbnail_path):
 
 def get_img_meta(img_id):
     try:
-        db = mysql()
-        db.connect()
-        sql = 'select * from image_data where img_is_delete = 0 and img_id = "%s"' % img_id
-        return db.query(sql)[0]
+        with ImageDao() as imageDao:
+            img_data = ImageDao.getImgData(imgId=img_id)
+        return img_data
```

Comment:
@stvreumi could you help me to add TODO comment for the future improvements as 
`#TODO Caller should check return value is not None`